src/main/java/org/zalando/nakadi/repository/EventTypeRepository.java

```diff
@@ -20,6 +20,8 @@
 
     void removeEventType(String name) throws InternalNakadiException, NoSuchEventTypeException;
 
+    void setEventTypeDeleted(String name) throws InternalNakadiException, NoSuchEventTypeException;
```

Comment:
From a client code perspective (invoking this method in a controller, for example) it's difficult to tell the difference from `removeEventType`. They seem to be the same thing. One would have to read the implementation to understand the difference. Also, why should we have true deletion and also "stored forever deletion" at the same time?